src/Abp.Web.Common/Abp.Web.Common.csproj

```diff
@@ -21,6 +21,8 @@
 
   <ItemGroup>
     <EmbeddedResource Include="Web\Localization\AbpWebXmlSource\*.xml" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
+    <None Remove="Web\Localization\AbpWebXmlSource\AbpWeb+ro.xml" />
```

Comment:
EmbeddedResource already has wildcards configured. So this and some of the following csproj configurations are not needed.

```

  <ItemGroup>
    <None Remove="Zero\Localization\Source\AbpZero-ro.xml" />
  </ItemGroup>

  <None Remove="Ldap\Localization\Source\AbpZero-ro.xml" />

  <ItemGroup>
    <None Remove="Zero\Localization\SourceExt\AbpZero-ro.xml" />
  </ItemGroup>
```

Reply:
👍 fix